src/StringHelper.php

```diff
@@ -729,6 +729,27 @@ public static function findBetweenLast(string $string, string $start, ?string $e
         return mb_substr($string, $startPos, $endPos - $startPos);
     }
 
+    /**
+     * Checks if a given string matches any of the provided patterns.
+     *
+     * Note that patterns should be provided without delimiters on both sides. For example, `te(s|x)t`.
```

Comment:
Isn't it better to include delimiters and flags per regex? Also, you'd be able to use better signature:

```php
matchAnyRegex(string $string, ...string $patterns): bool
```

Reply:
New method is a facade for `CombinedRegexp`, that requires separate patterns and flags, because he merges all patterns to one under the hood.